openmls/src/group/public_group/validation.rs

```diff
@@ -423,6 +423,11 @@ impl PublicGroup {
             if !self.treesync().is_leaf_in_tree(removed) {
```

Comment:
How do we handle transition from ValSem annotations to the dashboard? Shall we add both where applicable?

Reply:
https://github.com/openmls/validation.dhall/pull/14